server/src/main/java/org/opensearch/action/admin/cluster/remotestore/restore/RestoreRemoteStoreRequest.java

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
This file is added so that corresponding code in `RestoreService` is compiled without any errors. This file is same as defined here: https://github.com/opensearch-project/OpenSearch/pull/3576